src/translations/en/homepage.json

```
@@ -23,11 +23,14 @@
   "how.bloc4.title": "4. Publishing",
   "how.title": "How it works",
   "mission_statement": "When the services we use every day change their rules, we should be the first ones to know.",
+  "reuses.memos-elections-fr.author": "Reset",
+  "reuses.memos-elections-fr.subtitle": "Tracking platforms behaviour over electoral periods.",
```

All phrasings are in the active tense.
```suggestion
  "reuses.memos-elections-fr.subtitle": "Tracks major platforms behaviour over the French 2022 elections.",
```